resources/local/chainspec.toml.in

```diff
@@ -16,9 +16,8 @@ accounts_path = '${BASEDIR}/resources/local/accounts.csv'
 validator_slots = 5
 # Number of eras before an auction actually defines the set of validators. If you bond with a sufficient bid in era N, you will be a validator in era N + auction_delay + 1
 auction_delay = 3
-# The delay for the payout of funds, in eras. If a withdraw request is included in a block in era N
-# (other than the last one), they are paid out in the last block of era N + locked_funds_period.
-locked_funds_period = 15
+# Number of days (in milliseconds) after genesis that a genesis validator's bid is locked
+locked_funds_period_millis = 7776000000
```

Comment:
Maybe for the local one it should just be a few minutes, so it's easier to test?

Reply:
Set to 5 minutes.